### PR TITLE
SSL support

### PR DESCRIPTION
**Pull request by iisuslik43**

### CODE REVIEW

File: src/main/kotlin/com/bardsoftware/papeeria/backend/cosmas/CosmasServer.kt [hunk 1]
```diff
@@ -58,7 +70,12 @@ fun main(args: Array<String>) = mainBody {
     LOG.info("Try to bind in port ${arg.port}")
     val server =
             if (arg.bucket != "") {
-                CosmasServer(arg.port, CosmasGoogleCloudService(arg.bucket))
+                if(arg.certChain != null && arg.privateKey != null) {
```
Comment (dbarashev) [inline src/main/kotlin/com/bardsoftware/papeeria/backend/cosmas/CosmasServer.kt]: nit: space after `if`

File: src/main/kotlin/com/bardsoftware/papeeria/backend/cosmas/CosmasServer.kt [hunk 2]
```diff
@@ -20,18 +20,30 @@ import com.xenomachina.argparser.ArgParser
...
  * Simple server that will wait for request and will send response back.
  * It uses CosmasGoogleCloudService or CosmasInMemoryService to store files
  * @author Aleksandr Fedotov (iisuslik43)
  */
 class CosmasServer(port: Int, val service: CosmasGrpc.CosmasImplBase) {
-    private val server: Server = ServerBuilder
+
+    constructor(port: Int, service: CosmasGrpc.CosmasImplBase,
+                certChain: File, privateKey: File) : this(port, service) {
+        this.server = ServerBuilder
```
Comment (dbarashev) [inline src/main/kotlin/com/bardsoftware/papeeria/backend/cosmas/CosmasServer.kt]: Add Preconditions calls checking that files exist?

**Review by dbarashev — APPROVED** [review]
LGTM[2 comments]

File: src/main/kotlin/com/bardsoftware/papeeria/backend/cosmas/CosmasServer.kt (hunk 1 above)
Comment (iisuslik43) [reply src/main/kotlin/com/bardsoftware/papeeria/backend/cosmas/CosmasServer.kt]: done

File: src/main/kotlin/com/bardsoftware/papeeria/backend/cosmas/CosmasServer.kt (hunk 2 above)
Comment (iisuslik43) [reply src/main/kotlin/com/bardsoftware/papeeria/backend/cosmas/CosmasServer.kt]: done

**Pull request merged**